spyder/widgets/github/backend.py

```diff
@@ -14,9 +14,12 @@
 import logging
 import webbrowser
 
+import keyring
```

Comment:
Please enclose this in a try/except and make `keyring = None` if it fails.

Reply:
On a second thought, we don't need to make `keyring = None`. We need to `pass` in the `except` part.